src/common/transformations/src/transformations/utils/utils.cpp

```diff
@@ -167,6 +167,61 @@ std::shared_ptr<ngraph::Node> node_to_get_shape_value_of_indices_from_shape_sour
     return node_to_get_shape_value_of_indices_from_shape_node(shape_node, indices);
 }
 
+bool can_eliminate_eltwise_node(const std::shared_ptr<Node>& eltwise, const Output<Node>& constant, const Output<Node>& non_constant_input) {
+    if (!is_type<opset8::Add>(eltwise) &&
+        !is_type<opset8::Subtract>(eltwise) &&
+        !is_type<opset8::Multiply>(eltwise) &&
+        !is_type<opset8::Divide>(eltwise)) {
+        return false;
+    }
+
+    // check if constant has a single value with either 0 (for Add, Subtract) or 1 (for Multiply, Divide)
+    auto constant_ptr = std::dynamic_pointer_cast<opset8::Constant>(constant.get_node_shared_ptr());
+    if (!constant_ptr) {
+        return false;
+    }
+    float value;
+    if (!op::util::get_single_value(constant_ptr, value)) {
```

Comment:
Please try to use bitwise_identical field from Constant to check that all values are the same. Otherwise it will be very time consuming.